cmd/microcloud/main_init.go

```diff
@@ -537,6 +541,16 @@ func (c *initConfig) addPeers(sh *service.Handler) error {
 					}
 				}
 
+				mut.Lock()
```

Comment:
Do these locks need to be set within the reverter's func?

Reply:
Moving them into the revert hooks would mean they get evaluated when the `defer` executes. The reverts execute serially, so why would we move them into the hooks?

Reply:
Looked at it again. I thought you wanted to lock some variable but instead you lock the reverter itself so that you don't concurrently register the reverts.